torchgeo/datasets/chesapeake.py

```diff
@@ -22,11 +22,10 @@
 from matplotlib.figure import Figure
 from rasterio.crs import CRS
 from torch import Tensor
-
 from .geo import GeoDataset, RasterDataset
+from .nlcd import NLCD
```

Comment:
If you run `isort` on the file, it will automatically take care of these for you. See https://torchgeo.readthedocs.io/en/stable/user/contributing.html#linters